### ISSUE TITLE
关于预训练模型的训练方法的一个疑问

### ISSUE THREAD

**Issue by mmybit** [dailenson/SDT#81]
作者您好，我想询问您一下发布的预训练模型是只在 CASIA-OLHWDB (1.0-1.2)上训练吗，论文里写的在测试集 ICDAR-2013 抽80%训练是什么意思呀，是另一个实验设置吗？谢谢您！
To demonstrate the superiority of the proposed offline-to-offline handwriting generation framework, we use the offline character images of ICDAR-2013 competition database [54], which contains 60 writers and 3755 different Chinese characters for each writer. We
randomly select 80% of the entire dataset as the training set, and the remaining 20% as the test set.

**Comment by dailenson** [comment]
是的，您贴这段实验描述不是我们的主实验，是另一个拓展实验。我们的主实验做online手写生成的，我们也将其拓展到offline手写生成，您贴的这段就是offline手写生成的实验描述。主实验的描述请看正文的实验细节那里。
